XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -1972,6 +1972,24 @@ public enum CustomComboPreset
     [CustomComboInfo("Kyuten to Zanshin", "Replace Hissatsu: Kyuten with Zanshin when available.", SAM.JobID)]
     SamuraiKyutenZanshinFeature = 3421,
 
+    [IconsCombo([SAM.HissatsuKyuten, UTL.ArrowLeft, SAM.Iaijutsu])]
+    [SectionCombo("Kyuten")]
+    [ExpandedCustomCombo]
+    [CustomComboInfo("Shinten to Iaijutsu", "Replace Hissatsu: Shinten with Iaijutsu when available.", SAM.JobID)]
+    SamuraiShintenIaijutsuFeature = 3422,
```

Comment:
This is why it is crashing, this combo ID is now in use by `SamuraiIaijutsuSingleSenNoReplaceTsubameFeature`.